neo4j-test/doc-tests.js

```diff
@@ -41,69 +46,266 @@ describe('Tests for Documents', function () {
     await deleteAllNodesAndEdges();
   });
 
-  beforeEach('Create a dummy doc', async function () {
+  afterEach('Drop dummy Doc', async function () {
+    await dbFix.Delete(dbTables);
+  });
+
+  it('Add the elements of MAPK6-AKT1 dummy doc to Neo4j db', async function () {
     let loadTable = name => ({ rethink: r, conn: rdbConn, db: testDb, table: testDb.table(name) });
     let loadTables = () => Promise.all(dbTables.map(loadTable)).then(dbInfos => ({ docDb: dbInfos[0], eleDb: dbInfos[1] }));
 
     const { document } = await dbFix.Insert(fixture);
     const { docDb, eleDb } = await loadTables();
     const loadDocs = ({ id, secret }) => loadDoc({ docDb, eleDb, id, secret });
-    fixtureDocs = await Promise.all(document.map(loadDocs));
+    const fixtureDocs = await Promise.all(document.map(loadDocs));
+
+    let myDoc = fixtureDocs[0];
+
+    expect(await getNumNodes()).equal(0);
+    expect(await getNumEdges()).equal(0);
+    await addDocumentToNeo4j(myDoc);
+
+    let arrNodes = myDoc.elements().filter(ele => ele.isEntity());
+    let arrEdges = myDoc.elements().filter(ele => !(ele.isEntity()));
+
+    expect(await getNumNodes()).equal(2);
+    for (const n of arrNodes) {
+      let id = `${n.association().dbPrefix}:${n.association().id}`;
+      expect(await getGeneName(id)).equal(`${n.association().name}`);
+    }
+
+    expect(await getNumEdges()).equal(1);
+    for (const e of arrEdges) {
+      let edge = await getEdge(e.id());
+      expect(edge.properties.type).equal(e.type());
+      if (e.association().getSource()) {
+        expect(edge.properties.sourceId).equal(convertUUIDtoId(myDoc, e.association().getSource().id()));
+        expect(edge.properties.targetId).equal(convertUUIDtoId(myDoc, e.association().getTarget().id()));
+      } else {
+        expect(edge.properties.sourceId).equal(convertUUIDtoId(myDoc, e.elements()[0].id()));
+        expect(edge.properties.targetId).equal(convertUUIDtoId(myDoc, e.elements()[1].id()));
+      }
+      expect(edge.type).equal('INTERACTION');
```

Comment:
There might be some confusion with what `type` is. 

(a) Yes, and edge is of type `INTERACTION`
(b) The type of interaction, i.e. the biological mechanism of action between two proteins is `PHOSPHORYLATION`

So I thought we wanted (b), because (a) is sorta obvious.

Reply:
`edge.type` is `INTERACTION`, but 
`edge.properties.type` is `phosphorylation`

I can change it so that
`edge.type` is `PHOSPHORYLATION` (or whatever the type is) instead and get rid of the `edge.properties.type` field. 

Would that make more sense?
I would need to touch a few more files to make this change

Reply:
OK that's fine as is. I just wanted to know if the phosphorylation was there.